server/xpub-model/entities/file/index.js

```diff
@@ -25,23 +26,33 @@ class File extends BaseModel {
     }
   }
 
-  async save() {
+  async save(userId) {
     await this.$query().upsertGraphAndFetch(this)
 
+    const manuscript = await Manuscript.find(this.manuscriptId, userId)
```

Comment:
`File` object loading `Manuscript` objects on save, and saving them in turn? This looks like a lot of coupling between the two, as they are transactionally saved together. Thinking... :thinking: